### PR TITLE
- Created search view with icon 💻

### PR DESCRIPTION
**Pull request by MicahSphelele**

### CODE REVIEW

**Review by Gontse — CHANGES_REQUESTED** [review]
Any PR for UI Change needs to include a screenshot

**Comment by MicahSphelele** [comment]
> Any PR for UI Change needs to include a screenshot

Okay cool will do that for 

> _No description provided._

Simulator Screen Shot - iPod touch (7th generation) - 2021-06-12 at 00 23 04

**Comment by MicahSphelele** [comment]
> Any PR for UI Change needs to include a screenshot

I have attached the screenshot.

File: Recipe-App/Recipe-App/UIViews/RecipeListView.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//  Recipe-App
+//
+//  Created by Sphelele Ngubane on 2021/06/11.
+//
+
+import SwiftUI
+
+struct RecipeListView: View {
+    
+    @State var searchText: String = ""
```
Comment (Gontse) [inline Recipe-App/Recipe-App/UIViews/RecipeListView.swift:12]: shouldn't this be a binding property ?

Comment (MicahSphelele) [reply Recipe-App/Recipe-App/UIViews/RecipeListView.swift:12]: We can use the binding on another view that may want to use the searchText.

**Review by Gontse — APPROVED** [review]

**Pull request merged**